pkg/sql/schemachanger/scpb/element_generator.go

```diff
@@ -64,42 +64,39 @@ func run(in, out string) error {
 package scpb
 
 type ElementStatusIterator interface {
-	ForEachElementStatus(fn func(status, targetStatus Status, elem Element))
+	ForEachElementStatus(fn func(current Status, target TargetStatus, e Element))
 }
 {{ range . }}
 
 func (e {{ . }}) element() {}
 
 // ForEach{{ . }} iterates over elements of type {{ . }}.
 func ForEach{{ . }}(
-	b ElementStatusIterator, elementFunc func(status, targetStatus Status, element *{{ . }}),
+	b ElementStatusIterator, fn func(current Status, target TargetStatus, e *{{ . }}),
 ) {
   if b == nil {
     return
   }
-	b.ForEachElementStatus(func(status, targetStatus Status, elem Element) {
-		if e, ok := elem.(*{{ . }}); ok {
-			elementFunc(status, targetStatus, e)
+	b.ForEachElementStatus(func(current Status, target TargetStatus, e Element) {
+		if elt, ok := e.(*{{ . }}); ok {
+			fn(current, target, elt)
 		}
 	})
 }
 
 // Find{{ . }} finds the first element of type {{ . }}.
-func Find{{ . }}(b ElementStatusIterator) (currentStatus, targetStatus Status, element *{{ . }}) {
+func Find{{ . }}(b ElementStatusIterator) (current Status, target TargetStatus, element *{{ . }}) {
   if b == nil {
-    return currentStatus, targetStatus, element
+    return current, target, element
   }
-	b.ForEachElementStatus(func(cs, ts Status, elem Element) {
-		if element != nil {
-			return
```

Comment:
Yeah so this was really bad. I guess this didn't show up because we just don't use these `Find*` helpers that much. But, ouch.